source/_integrations/august.markdown

```diff
@@ -43,6 +43,7 @@ The `august` integration allows you to integrate your [August](https://august.co
 | Yale Assure Lock 2 | yes |
 | Yale Conexis L1 | yes |
 | Yale Conexis L2 | yes |
+| Yale Doorman L3 | yes |
 | Yale Linus | yes |
 | Yale Smart Safe | yes |
 
```

Comment:
Maybe add the line about untested here?

> Other devices not listed above have not been tested and may not function as expected.

Will cover your comment about them releasing new models.

Reply:
a6f6371d18e478b242bf3996042559e741005e7c